blog/intro-to-justfile.md

```
@@ -0,0 +1,346 @@
+```
+This allows me to call uxp from anywhere in the system and it will
+execute the setup\_infra recipe.
+
+### Interactive Mode
+
+Just has an interactive mode that allows you to select recipes from the
+list using the --choose flag. Another alias I like to use is:
+
+```{=bash}
+.j: aliased to just --justfile ~/dev/dotfiles/scripts/uxp-setup/justfile --working-directory ~/dev/dotfiles/scripts/uxp-setup --choose
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)